public/css/doctors.css

```diff
@@ -0,0 +1,100 @@
+*{
+    padding:0;
+    margin: 0;
+    font-family: lato;
+}
+main section .container{
+    width: 90%;
+    margin: 0 auto;
+    margin-top: 150px;
+}
+main section .container h1{
+    width: fit-content;
+    font-size: 40px;
+    margin: 0 auto;
+    border-bottom:1px #333 solid;
+    font-weight: 100;
+}
+main section .container .doctors{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    margin-top: 50px;
+}
+/******************** Doctor Style ***********************/
+main section .container .doctors .doctor{
+    width: 25%;
+    height: 600px;
+    margin-bottom: 50px;
+}
+main section .container .doctors .doctor .img{
+    width: 250px;
```

Comment:
advice: Avoid using too much descendant selectors
check [this](http://johnotander.com/css/2015/06/10/css-performance/)